### DISCUSSION TITLE
Primary category creation on Sunbird Platforms

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Knowlg/Community#55] [Q&A] [answered]
Hello All,

I have created a Primary Category on [dockstaging.sunbirded.org](http://dockstaging.sunbirded.org/) .

I am publishing the content from the dock staging creation portal , and trying to consume it on staging.sunbird.
But when I am accessing the content on staging.sunbirded.org I am getting an API error as :

`Entry is not found in external-store for object with identifier : obj-cat : primary category name`

I wanted to know is there a sync job from dock-staging to staging for primary category which is not working , or 
I need to create the Primary category on both the ends. 

Please guide me here.

cc: @coolbung @baljeetsingh @vaivk369 @pallakartheekreddy

**Comment by pallakartheekreddy** [comment, marked as answer]
@Xitija We don't have any sync job to sync primary categories created in dock staging. So, Whatever primary categories are created in dock staging we have to create them in sunbird staging as well.
